.github/workflows/go-ftw.yml

```diff
@@ -39,20 +39,21 @@ jobs:
       run: |
         go install github.com/fzipi/go-ftw@latest
         mkdir caddy
-        wget https://raw.githubusercontent.com/jptosso/coraza-caddy/master/caddy/main.go
+        wget https://gist.githubusercontent.com/jptosso/42dbde4835169371e54a351c95644370/raw/ca6f71573b1d7f1a295f452582951fd5d0724efd/main.go
         cp testdata/coreruleset/Caddyfile .
         ls -lh
         sed -i 's/\/\/ _ "github.com/_ "github.com/g' main.go
         mv main.go caddy/main.go
-        go get github.com/jptosso/coraza-caddy@ffa1472
+        go get github.com/corazawaf/coraza-caddy@8ed5dd2
         go mod tidy
         go get ./...
         go get -u github.com/jptosso/coraza-pcre@593842e
-        go install caddy/main.go
-        sudo setcap CAP_NET_BIND_SERVICE=+eip $(which caddy)
+        go build -o caddybin caddy/main.go
+        file caddybin
```

Comment:
```suggestion
```

Reply:
Presume this was for debugging

Reply:
Yes, I actually think it's useful to keep some commands like ls and file in github actions for future debugging. There were cases where the caddybin file wasn't created, so it's good to check for it